turbinia/turbiniactl.py

```diff
@@ -929,9 +929,13 @@ def process_evidence(
 
   if evidence_ and not args.force_evidence:
     if not config.SHARED_FILESYSTEM and evidence_.copyable:
+      # This is created so we can auto-upload files when they are copyable.
       if os.path.exists(evidence_.local_path):
         output_manager = OutputManager()
-        output_manager.setup(evidence_.type, request_id, remote_only=True)
+        # Passing in request_id as the uid in this case because we don't have an
+        # associated Task ID in this case.
```

Comment:
Small nit, but looks like `in this case` was mentioned twice

Reply:
Good catch, thanks!  Fixed.